src/main/java/com/google/api/codegen/transformer/MethodContext.java

```diff
@@ -14,14 +14,7 @@
  */
 package com.google.api.codegen.transformer;
 
-import com.google.api.codegen.config.ApiSource;
-import com.google.api.codegen.config.FlatteningConfig;
-import com.google.api.codegen.config.GapicProductConfig;
-import com.google.api.codegen.config.InterfaceConfig;
-import com.google.api.codegen.config.InterfaceModel;
-import com.google.api.codegen.config.MethodConfig;
-import com.google.api.codegen.config.MethodModel;
-import com.google.api.codegen.config.SingleResourceNameConfig;
+import com.google.api.codegen.config.*;
```

Comment:
Should reconfigure your IDE to not do this, I'm assuming you're using IntelliJ,

Editor > Code Style > Java

you can raise the `'*'` import settings to 9999 or something huge.

Reply:
oh thanks for the tip!
there are no more star imports anywhere in the codebase.